src/interfaces.ts

```diff
@@ -12,19 +17,23 @@ export interface IframeWrapperParams {
 export interface IWidgetThemeConfig {
   assets: {
     logo: {
-      dark: {
-        horizontal: string
-        vertical: string
-      }
-      light: {
-        horizontal: string
-        vertical: string
-      }
+      horizontal: string
+      vertical: string
     }
   }
   theme: Theme
 }
 
+export type AppInfo = {
```

Comment:
Let's stick to interfaces.

```suggestion
export interface IAppInfo = {
```